src/pages/notifications.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import PageTemplate from "@/components/templates/PageTemplate";
+import Notifications from "@/components/organisms/Notifications";
+
+export default function HistoryIndex() {
```

Comment:
Change the function name. It needs to be almost same as the filename.
`NotificationsIndex`?